### PR TITLE
docs(README): remove the "2" from Angular

### PR DESCRIPTION
**Pull request by juristr**
I wanted to support Igor's mission to remove the "2" from Angular on the web. Also the repo description would have to be adjusted. 😊

### CODE REVIEW

**Comment by Splaktar** [comment]
There is already widespread confusion about `angular-material` for Angular 1.x and `@angular/material`/`material2`/`angular-material2` for Angular 2+.

This spreads across StackOverflow and Google Groups where questions about Angular Material for Angular 2+ keep getting posted in the `angular-material` for Angular 1.x forum. Then tags in StackOverflow keep getting posted with just `angular-material` when the question is about `angular-material2` and Angular 2+ and not `angular-material` for Angular 1.x.

@jelbourn has already said that the [2 in the repo name isn't going away](https://github.com/angular/material2/issues/1517#issuecomment-254574712).

It seems like this PR does some to balance this by leaving the "for **Angular version 2 onwards**" text. But it seems like more needs to be done to disambiguate these repositories in the README.

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Material Design for Angular 2
+# Material Design for Angular
 [![npm version](https://badge.fury.io/js/%40angular%2Fmaterial.svg)](https://www.npmjs.com/package/%40angular%2Fmaterial)
 [![Build Status](https://travis-ci.org/angular/material2.svg?branch=master)](https://travis-ci.org/angular/material2)
 [![Gitter](https://badges.gitter.im/angular/material2.svg)](https://gitter.im/angular/material2?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-This is the home for the Angular team's Material Design components built on top of Angular 2.
+This is the home for the Angular team's Material Design components built for **Angular version 2 onwards**.
```
Comment (jelbourn) [inline README.md]: How about just "for Angular v2.0.0+"  ?

Comment (juristr) [inline README.md]: If I understood them properly, @igorminar as well as @StephenFluin said to me one should mention "version 2" and not just "v2" to make it more explicit. More correct even, "Angular" and "version" should not be in the same sentence. So it should maybe be "for version 2+". I just thought that would create confusion with material 1 & 2...

**Comment by IgorMinar** [comment]
I spoke with @jelbourn about the branding issue in the light of the #itsjustangular effort.

Unrelated to this PR, @juleskremer and I called for a meeting with Angular Devrel team to create a set of naming / branding guidelines for all things Angular. This will include how to deal with Angular 1 naming and branding (including angularjs.org) and other things like Angular Material.

This meeting will happen early next week when everyone is back in the office. Until then, let's hold any work on this until we get these guidelines. Otherwise we will likely need to redo this again in a few days. Sounds good?

**Comment by juristr** [comment]
:+1: of course :smiley:

**Comment by andrewseguin** [comment]
This change is a duplicate of the merged #2805

**Comment by Splaktar** [comment]
This has been causing quite a bit of confusion with my clients. They keep searching for Angular Material and finding material.angularjs.org and lots of references to AngularJS Material instead of Angular Material(2). A number of them have raised concerns about the library not working or wanting to consider another UI library because of these frustrations. So far I have been able to explain things, but it has certainly come at a cost of time and frustrated clients.

**Pull request closed without merging**